app/javascript/lib/validation/validate-file-content.js

```diff
@@ -417,8 +417,15 @@ function getValidationTrigger() {
   }
 }
 
+/** Trim messages logged to Mixpanel, to prevent HTTP 413 errors */
+function getTrimmedIssueMessages(issues) {
+  return issues.map(columns => {
+    return columns[2].slice(0, 200) // Show <= 200 characters for each message
```

Comment:
super nit-picky, but unless there are a large number of array-based callers, it's generally better to have functions accept singular entities, and let the callee handle iteration.  That way the logic is as isolated as possible for easy unit-testing

Reply:
Per Friday chat, we agree this is well-factored, and I've clarified the docstring in 67ad6f4.